packages/dd-trace/src/telemetry/index.js

```diff
@@ -286,11 +286,30 @@ function updateConfig (changes, config) {
   const application = createAppObject(config)
   const host = createHostObject()
 
-  const configuration = changes.map(change => ({
-    name: change.name,
-    value: Array.isArray(change.value) ? change.value.join(',') : change.value,
-    origin: change.origin
-  }))
+  const names = {
+    sampleRate: 'DD_TRACE_SAMPLE_RATE',
+    logInjection: 'DD_LOG_INJECTION',
+    headerTags: 'DD_TRACE_HEADER_TAGS',
+    tags: 'DD_TAGS'
+  }
```

Comment:
The config value names will be [normalized](https://github.com/DataDog/dd-go/blob/prod/trace/apps/tracer-telemetry-intake/telemetry-payload/static/config_norm_rules.json) when sent to the backend. 
`sampleRate` and `DD_TRACE_SAMPLE_RATE` map to the same thing, `logInjection` is accepted but `DD_LOG_INJECTION` won't be recognized. We need to create a PR to [config_norm_rules ](https://github.com/DataDog/dd-go/blob/prod/trace/apps/tracer-telemetry-intake/telemetry-payload/static/config_norm_rules.json) so that `headerTags` will be accepted. `DD_TAGS` will be accepted but `tags` won't be recognized. Since we will be sending the other config values to the backend with camel case names, I'm suggesting we do that here too, and create a PR to the json file to add the missing mappings.

Reply:
Not sure I understand. Are you saying that the value that is literally used by the backend to send the information to us will not be recognized by the same backend when reported as telemetry? If that's the case it sounds like a bug in the backend that should be fixed, and thus I don't think it's on us to change anything in another PR. The safe approach is to send the same names we received.

Reply:
I asked about this and was told this is all still in flight and not required to be correct for now. System tests will be added when we know exactly the values that need to be sent.